docs: add 1.15 release-notes (#9046)
microsoft/playwright: docs/src/release-notes-csharp.md

<!-- TOC -->

## Version 1.15

> ### 🖱️ Mouse wheel emulation

Reviewer: ```suggestion
### 🖱️ Mouse Wheel
```

By using [`Page.Mouse.WheelAsync`](https://playwright.dev/dotnet/docs/next/api/class-mouse#mouse-wheel) you are now able to scroll vertically or horizontally.

> ### 📜 New Header API

Reviewer: ```suggestion
### 📜 New Headers API
```

> Previously it was not possible to get multiple header values of a response. This is now  possible and additional helper functions were added:

Reviewer: ```suggestion
Previously it was not possible to get multiple header values of a response. This is now possible and additional helper functions are available:
```

- [Request.AllHeadersAsync()](https://playwright.dev/dotnet/docs/next/api/class-request#request-all-headers)
- [Request.HeadersArrayAsync()](https://playwright.dev/dotnet/docs/next/api/class-request#request-headers-array)
- [Request.HeaderValueAsync(name: string)](https://playwright.dev/dotnet/docs/next/api/class-request#request-header-value)
- [Response.AllHeadersAsync()](https://playwright.dev/dotnet/docs/next/api/class-response#response-all-headers)
- [Response.HeadersArrayAsync()](https://playwright.dev/dotnet/docs/next/api/class-response#response-headers-array)
- [Response.HeaderValueAsync(name: string)](https://playwright.dev/dotnet/docs/next/api/class-response#response-header-value)
- [Response.HeaderValuesAsync(name: string)](https://playwright.dev/dotnet/docs/next/api/class-response/#response-header-values)

### 🌈 Forced-Colors emulation

Its now possible to emulate the `forced-colors` CSS media feature by passing it in the [context options](https://playwright.dev/dotnet/docs/next/api/class-browser#browser-new-context-option-forced-colors) or calling [Page.EmulateMediaAsync()](https://playwright.dev/dotnet/docs/next/api/class-page#page-emulate-media).

### New APIs

> - [Page.RouteAsync()](https://playwright.dev/dotnet/docs/next/api/class-page#page-route) accepts now a `times` option to specify how often a route should be matched.

Reviewer: ```suggestion
- [Page.RouteAsync()](https://playwright.dev/dotnet/docs/next/api/class-page#page-route) accepts new `times` option to specify how many times this route should be matched.
```
